sdk/ai/azure-ai-inference/README.md

```diff
@@ -145,14 +145,84 @@ client.completeStream(new ChatCompletionsOptions(chatMessages))
 
-<!--
+### Chat with image URL
+
+```java readme-sample-chatWithImageUrl
+List<ChatMessageContentItem> contentItems = new ArrayList<>();
+contentItems.add(new ChatMessageTextContentItem("Describe the image."));
+contentItems.add(new ChatMessageImageContentItem(
+    new ChatMessageImageUrl("<URL>")));
+
+List<ChatRequestMessage> chatMessages = new ArrayList<>();
+chatMessages.add(new ChatRequestSystemMessage("You are a helpful assistant."));
+chatMessages.add(ChatRequestUserMessage.fromContentItems(contentItems));
+
+ChatCompletions completions = client.complete(new ChatCompletionsOptions(chatMessages));
+
+for (ChatChoice choice : completions.getChoices()) {
+    System.out.printf("%s.%n", choice.getMessage().getContent());
+}
+```
+For a complete sample example, see sample [Image URL][sample_chat_with_image_url].
+
+### Chat with image file
+
+```java readme-sample-chatWithImageFile
+Path testFilePath = Paths.get("<path-to-image-file>");
+List<ChatMessageContentItem> contentItems = new ArrayList<>();
+contentItems.add(new ChatMessageTextContentItem("Describe the image."));
+contentItems.add(new ChatMessageImageContentItem(testFilePath, "<image-format>"));
+
+List<ChatRequestMessage> chatMessages = new ArrayList<>();
+chatMessages.add(new ChatRequestSystemMessage("You are a helpful assistant."));
+chatMessages.add(ChatRequestUserMessage.fromContentItems(contentItems));
+
+ChatCompletions completions = client.complete(new ChatCompletionsOptions(chatMessages));
+
+for (ChatChoice choice : completions.getChoices()) {
+    System.out.printf("%s.%n", choice.getMessage().getContent());
+}
+```
+For a complete sample example, see sample [Image File][sample_chat_with_image_file].
+
 ### Text embeddings
 
 ```java readme-sample-getEmbedding
+EmbeddingsClient client = new EmbeddingsClientBuilder()
+    .endpoint("{endpoint}")
+    .credential(new AzureKeyCredential("{key}"))
+    .buildClient();
+
+List<String> promptList = new ArrayList<>();
+String prompt = "Tell me 3 jokes about trains";
+promptList.add(prompt);
+
+EmbeddingsResult embeddings = client.embed(promptList);
+
+for (EmbeddingItem item : embeddings.getData()) {
+    System.out.printf("Index: %d.%n", item.getIndex());
+    for (Float embedding : item.getEmbeddingList()) {
+        System.out.printf("%f;", embedding);
+    }
+}
 ```
 For a complete sample example, see sample [Embedding][sample_get_embedding].
 
--->
+### Function calls
+
+For a complete sample example, see sample [Function Calls][sample_function_calls].
+
+### Streaming function calls
+
+For a complete sample example, see sample [Streaming Function Calls][sample_streaming_function_calls].
+
+### Get Model information
+
+```java readme-sample-getModelInfo
+ModelInfo modelInfo = client.getModelInfo();
+
+System.out.printf("modelName: %s, modelNameProvider: %s", modelInfo.getModelName(), modelInfo.getModelProviderName());
+```
```

Comment:
Why not print the 3rd value in modelInfo?

Reply:
[Updated](https://github.com/Azure/azure-sdk-for-java/pull/42008/commits/1d40edd54fb4809dd161eaf7b8cdab719f74a6e1)